vlasiator.cpp

```diff
@@ -109,9 +111,9 @@ void addTimedBarrier(string name){
    MPI_Barrier(MPI_COMM_WORLD);
 }
 
-void computeNewTimeStep(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
-			FsGrid< fsgrids::technical, FS_STENCIL_WIDTH> & technicalGrid, Real &newDt, bool &isChanged) {
-
+void computeNewTimeStep(dccrg::Dccrg<SpatialCell, dccrg::Cartesian_Geometry>& mpiGrid,
+                        std::span<fsgrids::technical> technical, FieldSolverGrid &fsgrid, Real& newDt,
```

Comment:
Taking this one as an example (I now only looked through this file) – I would find it more logical to have fsgrid first, then the views. But that'll be a tedious change of now instead consistently fsgrid is always last in such lists. So if the Vlasiator team is fine with this, let's go with it and go into "proper" things to do here in this PR.